yb-voyager/cmd/analyzeSchema.go

```diff
@@ -61,82 +90,78 @@ var (
-	withOidsRegex         = regexp.MustCompile(`(?i)CREATE TABLE (IF NOT EXISTS )?([a-zA-Z0-9_."]+) .*WITH OIDS`)
-	intvlRegex            = regexp.MustCompile(`(?i)CREATE TABLE (IF NOT EXISTS )?([a-zA-Z0-9_."]+) .*interval PRIMARY`)
+	createConvRegex       = re("CREATE", opt("DEFAULT"), "CONVERSION", capture(ident))
+	alterConvRegex        = re("ALTER", "CONVERSION", capture(ident))
+	gistRegex             = re("CREATE", "INDEX", ifNotExists, capture(ident), "ON", capture(ident), anything, "USING GIST")
+	brinRegex             = re("CREATE", "INDEX", ifNotExists, capture(ident), "ON", capture(ident), anything, "USING brin")
+	spgistRegex           = re("CREATE", "INDEX", ifNotExists, capture(ident), "ON", capture(ident), anything, "USING spgist")
+	rtreeRegex            = re("CREATE", "INDEX", ifNotExists, capture(ident), "ON", capture(ident), anything, "USING rtree")
+	ginRegex              = re("CREATE", "INDEX", ifNotExists, capture(ident), "ON", capture(ident), anything, "USING GIN", capture(commaSeperatedStrings))
+	viewWithCheckRegex    = re("VIEW", capture(ident), anything, "WITH", "CHECK", "OPTION")
+	rangeRegex            = re("PRECEDING", "and", anything, ":float")
+	fetchRegex            = re("FETCH", anything, "FROM")
+	fetchRelativeRegex    = re("FETCH", "RELATIVE")
+	backwardRegex         = re("MOVE", "BACKWARD")
+	fetchAbsRegex         = re("FETCH", "ABSOLUTE")
+	alterAggRegex         = re("ALTER", "AGGREGATE", capture(ident))
+	dropCollRegex         = re("DROP", "COLLATION", ifExists, capture(commaSeperatedStringsMoreThan1))
+	dropIdxRegex          = re("DROP", "INDEX", ifExists, capture(commaSeperatedStringsMoreThan1))
+	dropViewRegex         = re("DROP", "VIEW", ifExists, capture(commaSeperatedStringsMoreThan1))
+	dropSeqRegex          = re("DROP", "SEQUENCE", ifExists, capture(commaSeperatedStringsMoreThan1))
+	dropForeignRegex      = re("DROP", "FOREIGN", "TABLE", ifExists, capture(commaSeperatedStringsMoreThan1))
+	dropIdxConcurRegex    = re("DROP", "INDEX", "CONCURRENTLY", ifExists, capture(ident))
+	trigRefRegex          = re("CREATE", "TRIGGER", capture(ident), anything, "REFERENCING")
+	constrTrgRegex        = re("CREATE", "CONSTRAINT", "TRIGGER", capture(ident))
+	currentOfRegex        = re("WHERE", "CURRENT", "OF")
+	amRegex               = re("CREATE", "ACCESS", "METHOD", capture(ident))
+	idxConcRegex          = re("REINDEX ", anything, " ", capture(ident))
+	storedRegex           = re(capture(normalIdent), capture(normalIdent), "GENERATED", "ALWAYS", anything, "STORED")
+	partitionColumnsRegex = re("CREATE", "TABLE", ifNotExists, capture(ident), `\(`+capture(commaSeperatedStrings)+`\) PARTITION BY`, capture("[A-Za-z]+"), `\(`, capture(commaSeperatedStrings), `\)`)
+	likeAllRegex          = re("CREATE", "TABLE", ifNotExists, capture(ident), anything, "LIKE", anything, "INCLUDING ALL")
+	likeRegex             = re("CREATE", "TABLE", ifNotExists, capture(ident), anything, `\(`, "LIKE")
```

Comment:
Yes, like in this case `re("CREATE", "TABLE", ifNotExists, capture(ident), anything, `\(`, "LIKE")` if I didn't escape it will ask to add a closing brace but we want it to just match as brace in the SQL query.